lib/puppet/provider/group/directoryservice.rb

```diff
@@ -9,4 +9,10 @@
   confine :operatingsystem => :darwin
   defaultfor :operatingsystem => :darwin
   has_feature :manages_members
+
+  def members_insync?(current, should)
+  	return false unless current
+	return current.sort == should.sort
```

Comment:
Can `should` contain duplicates? On Windows it can, so we do https://github.com/puppetlabs/puppet/blob/master/lib/puppet/provider/group/windows_adsi.rb#L33